plugins/gameobjects/blitter/blitterbase/BlitterBase.d.ts

```diff
@@ -64,11 +67,11 @@ declare class BlitterBase extends Phaser.GameObjects.GameObject {
     blendMode: Phaser.BlendModes | string;
     setBlendMode(value: string | Phaser.BlendModes): this;
 
+    width: number;
+    height: number;
```

Comment:
`width` and `height` are described here:

https://github.com/photonstorm/phaser/blob/master/src/gameobjects/components/ComputedSize.js#L28
https://github.com/photonstorm/phaser/blob/master/src/gameobjects/components/ComputedSize.js#L41